src/sentry/static/sentry/app/components/actions/actionLink.tsx

```diff
@@ -47,16 +45,22 @@ export default class ActionLink extends React.Component {
       );
     } else {
       return (
-        <a
+        <ActionLinkAnchor
           data-test-id={testId}
           aria-label={title}
           className={classNames(className, {disabled})}
           onClick={disabled ? undefined : onAction}
           disabled={disabled}
         >
           {children}
-        </a>
+        </ActionLinkAnchor>
       );
     }
   }
 }
+
+const ActionLinkAnchor = styled('a')<{
+  disabled?: boolean;
+}>`
+  pointer-events: none;
+`;
```

Comment:
Doesn't look like you're using the `disabled` prop in here?

Reply:
Ah good catch.

Reply:
Fixed but Github is locking on the wrong line of code.